lib/Ed25519KeyPair.js

```diff
@@ -341,15 +434,23 @@ function ed25519VerifierFactory(key) {
 }
 
 /**
- * Wrap Base58 decoding operations in order to provide consistent error
- * messages.
- *
- * @param {function} decode - the decode function to use.
- * @param {string} keyMaterial - the Base58 encoded key material to decode.
- * @param {string} type - a description of the keyMaterial that will be included
- *   in an error message (e.g. 'public', 'private').
+ * @example
```

Comment:
It's nice to have internal docs for funcs, but I don't think this should be exposed in the public docs.  An `@ignore` would fix that.

Reply:
- [x] add @ignore to _base58Decode function